doc/man/lttng-enable-event.1.txt

```diff
@@ -127,11 +137,10 @@ application's execution, only one event is emitted.
 
 The available conditions for the Linux kernel domain are:
 
-* Tracepoint/system call name ('EVENT' argument with
-  option:--tracepoint or option:--syscall options) or
-  dynamic probe/function name/address
-  (option:--probe or option:--function option's argument) which must
-  match event source's equivalent.
+* Tracepoint/system call name ('EVENT' argument with option:--tracepoint
+  or option:--syscall options) or dynamic probe/function name/address
+  (option:--probe, option:--userspace-probe, and option:--function
+  option's argument) which must match event source's equivalent.
```

Comment:
Is there a word missing on the last line?

Reply:
event source's equivalent [tracepoint/system call name] (implicit).